plugins/plugin-chart-pivot-table/src/plugin/controlPanel.ts

```diff
@@ -0,0 +1,250 @@
+                'Minimum',
+                'Maximum',
+                'First',
+                'Last',
+                'Sum as Fraction of Total',
+                'Sum as Fraction of Rows',
+                'Sum as Fraction of Columns',
+                'Count as Fraction of Total',
+                'Count as Fraction of Rows',
+                'Count as Fraction of Columns',
+              ]),
+              default: 'Sum',
+              description: t(
+                'Aggregate function to apply when pivoting and computing the total rows and columns',
+              ),
+              renderTrigger: true,
+            },
+          },
+        ],
+        [
+          {
+            name: 'transposePivot',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Transpose pivot'),
+              default: false,
+              description: t('Swap rows and columns'),
+              renderTrigger: true,
+            },
+          },
+        ],
+      ],
+    },
+    {
+      label: t('Options'),
+      expanded: true,
+      controlSetRows: [
+        [
+          {
+            name: 'valueFormat',
+            config: { ...sharedControls.y_axis_format, label: t('Value format') },
+          },
+        ],
+        [
+          {
+            name: 'tableRenderer',
+            config: {
+              type: 'SelectControl',
+              label: t('Pivot table type'),
+              default: 'Table With Subtotal',
+              choices: [
+                // [value, label]
+                ['Table With Subtotal', t('Table')],
+                ['Table With Subtotal Heatmap', t('Table Heatmap')],
+                ['Table With Subtotal Col Heatmap', t('Table Col Heatmap')],
+                ['Table With Subtotal Row Heatmap', t('Table Row Heatmap')],
+              ],
+              renderTrigger: true,
+              description: t('The type of pivot table visualization'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'rowOrder',
+            config: {
+              type: 'SelectControl',
+              label: t('Rows sort by'),
+              default: 'key_a_to_z',
+              choices: [
+                // [value, label]
+                ['key_a_to_z', t('key alphabetically')],
+                ['value_a_to_z', t('value ascending')],
+                ['value_z_to_a', t('value descending')],
+              ],
+              renderTrigger: true,
+              description: t('Order of rows'),
+            },
+          },
+          {
+            name: 'colOrder',
+            config: {
+              type: 'SelectControl',
+              label: t('Cols sort by'),
+              default: 'key_a_to_z',
+              choices: [
+                // [value, label]
+                ['key_a_to_z', t('key alphabetically')],
+                ['value_a_to_z', t('value ascending')],
+                ['value_z_to_a', t('value descending')],
+              ],
```

Comment:
Should we also support `key_z_to_a`?

Reply:
Done. It's not supported in the original `react-pivottable`, I implemented it in our fork `@superset-ui/react-pivottable`